sdk/src/main/java/com/google/cloud/dataflow/sdk/io/CompressedSource.java

```diff
@@ -284,7 +284,7 @@ public void validate() {
    * from the requested file name that the file is not compressed.
    */
   @Override
-  protected final boolean isSplittable() throws Exception {
+  protected final boolean isSplittable() {
```

Comment:
Problem: this is a backwards-incompatible change I have to fix in DataflowJavaSDK